lib/org/openpsa/relatedto/plugin.php

```diff
@@ -31,8 +31,8 @@ class org_openpsa_relatedto_plugin extends midcom_baseclasses_components_plugin
      */
     public static function create($from_obj, $from_component, $to_obj, $to_component, $status = false, $extra = false)
     {
-        if (   !is_object($from_obj)
-            || !is_object($to_obj)) {
+        if (   !is_object($from_obj)|| !is_object($to_obj)) 
+        {
```

Comment:
coding style: Brackset on the wrong line, too many whitespaces in the beginngn of conditions